assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -179,6 +179,24 @@ export default class LiveEditor {
     this.view.focus();
   }
 
+  /**
+   * Focuses the editor in given line.
+   *
+   */
+  focusLine(lineNumber) {
+    setTimeout(() => {
+      const line = this.view.state.doc.line(lineNumber);
+
+      this.view.dispatch({
+        selection: { anchor: line.from },
+        scrollIntoView: EditorView.scrollIntoView(this.view.contentDOM, {
+          scrollMode: "if-needed",
+          behavior: "smooth",
+        }),
+      });
+    }, 300);
```

Comment:
Because it doesn't scroll into the code cell smoothly after clicking to go to definition.
I tested and with a big notebook and it doesn't work well, so I'll need to improve.

Perhaps, listen to some event?

Reply:
Once you update the handling to `#go-to-definition`, I can have a look!

Reply:
@aleDsz I removed the unused code. I also removed both `setTimeout` and scrolling here, and it works as expected for me (tested in a few browsers). If you can reproduce any scrolling issues, let me know!